File: src/main/java/no/nav/service/pensjon/regler/to/BeregnAlderspensjon2025ForsteUttakResponse.java
package no.nav.service.pensjon.regler.to;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning2011.BeregningsResultatAlderspensjon2025;

public class BeregnAlderspensjon2025ForsteUttakResponse extends ServiceResponse {

	private BeregningsResultatAlderspensjon2025 beregningsResultat;
	private BeregningsResultatAlderspensjon2025 beregningsResultatTilRevurdering;
	private Pakkseddel pakkseddel;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnAlderspensjon2025ForsteUttakResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning2011.BeregningsResultatAlderspensjon2025


class BeregnAlderspensjon2025ForsteUttakResponse : ServiceResponse() {
    var beregningsResultat: BeregningsResultatAlderspensjon2025? = null
    var beregningsResultatTilRevurdering: BeregningsResultatAlderspensjon2025? = null
}